swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
 class Item:
-    pass
\ No newline at end of file
+    """Parent class of Clothing, Decor, and Electronics"""
+
+    # Wave 2, create category attribute
+    # Wave 5, create condition attribute
+    def __init__(self, category = None, condition = None, age = None):
+        """category and condition are keyword arguments that optionally pass in."""
+        if not category:
+            category = ""
+        if not condition:
+            condition = 0
+        if not age:
+            age = 0
+        self.category = category
+        self.condition = condition
+        self.age = age
+        
```

Comment:
It would be ok to have the default arguments in the function signature in this case. We could do:

```
def __init__(self, category="", condition=0, age=0):
```

This is OK because strings and ints are immutable.